src/main/kotlin/com/devfloor/untitled/article/domain/ArticleRepository.kt

```diff
@@ -0,0 +1,6 @@
+package com.devfloor.untitled.article.domain
+
+import org.springframework.data.jpa.repository.JpaRepository
+
+interface ArticleRepository : JpaRepository<Article, Long> {
+}
```

Comment:
이게 자바가 아니라 코틀린이라 코드 본문을 사용하지 않을 경우 `{}`이거 지워도 되더라고. 우리 안쓰이는 곳은 다 지우면 좋을 듯?

```suggestion
interface ArticleRepository : JpaRepository<Article, Long>
```

Reply:
근데 이런건 나중에 prettier 쓰게되면 가능한부분이니까 지금은 일단 신경안써도되지않아? 일단 커밋 반영 해놓긴할게 !